<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformSliders</class>
 <widget class="QWidget" name="TransformSliders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>241</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>937</width>
    <height>241</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>937</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(67, 67, 67);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Object Rotation</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Object Scale</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Object Location</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(85, 85, 85);
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="LCDSlider" name="xRot" native="true"/>
        </item>
        <item>
         <widget class="LCDSlider" name="yRot" native="true"/>
        </item>
        <item>
         <widget class="LCDSlider" name="zRot" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="clearRot">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
background-color: rgb(175, 175, 175);
border-style: inset;
border-radius: 2px;
text-decoration: none;
color: rgb(0, 0, 0);
}
QPushButton:hover
{
background-color: rgb(235, 235, 235);
border-style: inset;
}
</string>
          </property>
          <property name="text">
           <string>Clear Rotation</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="LCDSlider" name="xScale" native="true"/>
        </item>
        <item>
         <widget class="LCDSlider" name="yScale" native="true"/>
        </item>
        <item>
         <widget class="LCDSlider" name="zScale" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="clearScale">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
background-color: rgb(175, 175, 175);
border-style: inset;
border-radius: 2px;
text-decoration: none;
color: rgb(0, 0, 0);
}
QPushButton:hover
{
background-color: rgb(235, 235, 235);
border-style: inset;
}
</string>
          </property>
          <property name="text">
           <string>Clear Scale</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="LCDSlider" name="xPos" native="true"/>
        </item>
        <item>
         <widget class="LCDSlider" name="yPos" native="true"/>
        </item>
        <item>
         <widget class="LCDSlider" name="zPos" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="clearPos">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
background-color: rgb(175, 175, 175);
border-style: inset;
border-radius: 2px;
text-decoration: none;
color: rgb(0, 0, 0);
}
QPushButton:hover
{
background-color: rgb(235, 235, 235);
border-style: inset;
}
</string>
          </property>
          <property name="text">
           <string>Clear Location</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LCDSlider</class>
   <extends>QWidget</extends>
   <header>src/widgets/lcdslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
